prompts/npm/scripts/payload.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Oh wait, this is not ready to merge yet is it?  This is still the version with hard coded node roots.

Reply:
This is now ready for review.